### PR TITLE
Set up docs container

### PR DESCRIPTION
**Pull request by angela-tran**
## What this PR does
 - Set up configuration to build a `docs` container that starts up `mkdocs`
 - Copy latest API specification documentation from [eligibility-server/docs@8223615](https://github.com/cal-itp/eligibility-server/tree/8223615554cdecfaf7f56049e9dbe8f59a42dde5/docs) into a `docs` folder

### CODE REVIEW

File: .devcontainer/docker-compose.yml [hunk 1]
```diff
@@ -3,18 +3,18 @@ version: "3.8"
 services:
   dev:
     build:
-      context: .
-      dockerfile: Dockerfile
+      context: ..
+      dockerfile: /.devcontainer/Dockerfile
```
Comment (thekaveman) [inline .devcontainer/docker-compose.yml:7]: Probably just a typo, the `/` at the beginning is making VS Code look in the root of the filesystem for a .devcontainer/Dockerfile 😅 

Should be just `.devcontainer/Dockerfile`

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+# API Specification
```
Comment (thekaveman) [inline docs/README.md:1]: I think we will want the first docs page to be a little more general / about the library, maybe some quick usage samples, etc.

Let's reorg this file into `docs/specification/README.md`

File: docs/example-transactions.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
+# Example transactions
```
Comment (thekaveman) [inline docs/example-transactions.md:1]: See comment above. Let's reorg this page into `docs/specification/example-transactions.md` or even just `examples.md`.

File: docs/token-signing-encryption.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Token signing and encryption
+
+The Eligiblity Verification API makes use of Signed and Encrypted JSON Web Tokens (JWS, JWE, JWT) as a means of data transfer.
+
+A public/private keypair must be generated by each party (Benefits Client and Eligibility Verification Server).
```
Comment (thekaveman) [inline docs/token-signing-encryption.md:5]: Let's make this more generic here:

> A public/private keypair must be generated by each party (Client and Server).

File: docs/token-signing-encryption.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+[Example keys](https://github.com/cal-itp/benefits/tree/dev/localhost/keys) are included for the test verification server
```
Comment (thekaveman) [inline docs/token-signing-encryption.md:6]: Expand to point to example keys in both `benefits` and `eligibility-server`

File: docs/token-signing-encryption.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Extract the public key
+
+```bash
+openssl rsa -in [private key created above].key -pubout > [file name].pub
+```
+
+There are two new files:
+
+* `[file name].key`: private key in PEM format, apply to a `TransitAgency` instance in Django
```
Comment (thekaveman) [inline docs/token-signing-encryption.md:25]: Again we can make this a little more generic: 

> private key in PEM format, needed by the Client only

File: docs/token-signing-encryption.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+* `[file name].pub`: public key in PEM format, give to the Eligibility Verification server
+
+A public key in PEM format from the Eligibility Verification server is also required, and must be applied to an
+`EligiblityVerifier` instance in Django.
```
Comment (thekaveman) [inline docs/token-signing-encryption.md:29]: > The Client also requires a public key from the Eligibility Verification server, so this process must be repeated to generate the _Server's_ keypair.
>
> The _Server's_ private key should not be shared.

**Review by thekaveman — CHANGES_REQUESTED** [review]
Awesome! I was able to get all this running locally after the tiny change to the Compose file (see below).

I requested some minor reorganization of the content in anticipation of having usage/code docs in addition to the specification.

File: docs/README.md (hunk 2 above)
Comment (angela-tran) [reply docs/README.md]: Created https://github.com/cal-itp/eligibility-api/issues/9 for this

**Comment by angela-tran** [comment]
Thanks for the feedback @thekaveman . I pushed new changes - let me know if anything looks off.

**Review by thekaveman — APPROVED** [review]
📖 📑 📚

**Pull request merged**